src/tribler-gui/tribler_gui/widgets/tablecontentdelegate.py

```diff
@@ -264,9 +264,16 @@ def draw_subscribed_control(self, painter, option, index, data_item):
         if data_item[u'state'] == u'Personal':
             return True
 
-        self.subscribe_control.paint(
-            painter, option.rect, index, toggled=data_item.get('subscribed'), hover=index == self.hover_index
-        )
+        # When cursor leaves the table, we must "forget" about the button_box
+        if self.hoverrow == -1:
```

Comment:
Nit: I always prefer to keep magic numbers in constants and can only recommend you do the same ;)